docs/source/user-guide/kubernetes.md

```diff
@@ -65,13 +65,13 @@ metadata:
 spec:
   type: Scala
   mode: cluster
-  image: ghcr.io/apache/datafusion-comet:spark-3.4-scala-2.12-0.2.0
+  image: ghcr.io/apache/datafusion-comet:spark-3.4-scala-2.12-0.5.0
```

Comment:
This should really be 0.6.0 but the Docker image failed to publish. I filed https://github.com/apache/datafusion-comet/issues/1417